packages/core/src/metadata.ts

```diff
@@ -73,6 +166,13 @@ class Metadata implements IMetadata {
     };
   };
 
+  tss?: {
```

Comment:
Maybe we should make this entry depend on `tssTag` directly instead of making each entry of `TssMetadata` depend on `tssTag`?

Reply:
good suggestion